samples/system-test/topics.test.js

```diff
@@ -188,6 +189,23 @@ describe('topics', () => {
     );
   });
 
+  it('should resume publish', async () => {
+    const topics = require('../resumePublish');
```

Comment:
This is an interesting outlier in the design of the sample tests. Most of the rest of them will just spawn another copy of node to run them, which also tests the command line handling. I think this is probably fine, but I thought I'd note it.

Reply:
It turns out that there is a test in subscription.test.js, which is where I copied from, that does this. I have changed it to be consistent with the rest of the tests in this file.